creds/benches/proof_benchmark.rs

```diff
@@ -108,9 +108,11 @@ pub fn show_bench(c: &mut Criterion) {
 
     let mut client_state = ClientState::<Bn254>::new(
         public_inputs.clone(),
+        None,
         proof.clone(),
         vk.clone(),
         pvk.clone(),
+        "empty config.json".to_string()
```

Comment:
If we need to specify this, would we use "empty config.json"? Sounds implementation specific. Use something more generic instead, e.g., "empty config"?

Reply:
I changed it to "empty configuration string", but this is an internal API that we're calling here for benchmarking, so I don't think it matters too much.